CHANGELOG.md

```diff
@@ -1,5 +1,33 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2023-10-22
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+
+* Kubernetes 1.25 is being deprecated at the end of January 2024 and support will transition to our [platform support policy](https://learn.microsoft.com/azure/aks/supported-kubernetes-versions?tabs=azure-cli#platform-support-policy).
+* No new clusters can be created with [Azure AD Integration (legacy)](https://learn.microsoft.com/azure/aks/azure-ad-integration-cli). Existing AKS clusters with Azure Active Directory integration will keep working. All Azure AD Integration (legacy) AKS clusters will be migrated to [AKS-managed Azure AD](https://learn.microsoft.com/azure/aks/managed-azure-ad) automatically starting from 1st Dec. 2023. We recommend updating your cluster with AKS-managed Azure AD before 1 Dec 2023. This way you can manage the API server downtime during non-business hours.
+
+### Release notes 
+* Features
+  * Added RBAC permissions to CNS for handling swiftv2 CRD resources("multitenantpodnetworkconfigs", "podnetworkinstances" and "podnetworks"). Note that swiftv2 flow is still under development and is not done yet. This change was done to unblock swiftv2 development.
+  * Omsagent addon can now be interacted with using helm commands. The default value is set to true for omsagent addon v2 migration. 
+  * The default Fleet agent is now set to the new version (v1beta1), supporting the latest release in [Fleet](https://github.com/Azure/fleet).
+  * Bumped dropgz to v0.0.11 for k8s 1.28.0+.
```

Comment:
Component update

Reply:
Talked to engineer and he confirmed it is a bug fix. I added additional description from him.